---
layout: subsite-galaxy
---

<br/>

# Welcome to **Galaxy Proteomics**
{:.no_toc}

 | 
--- | ---
<img src="/assets/media/GalaxyP_logo.png" height="300px" width="300px" alt="GalaxyP logo"/><br/> *This European Galaxy branches off from the main<br/>[usegalaxy.eu](https://usegalaxy.eu){:target="_blank"} server and focuses on one specific domain:<br/>Proteomics.* **Galaxy Proteomics** is a multiple 'omics'<br/>data analysis platform with particular emphasis on mass<br/>spectrometry-based proteomics. It is hosted by<br/>GalaxyEU and maintained by the [GalaxyP team](http://galaxyp.org){:target="_blank"} from the<br/>University of Minnesota, MN, USA. |<a class="twitter-timeline" data-width="900" data-height="450" href="https://twitter.com/usegalaxyp?ref_src=twsrc%5Etfw">Tweets by usegalaxyp</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
{:.table}


# Content
{:.no_toc}

1. TOC
{:toc}

<br/>

# Tutorials

Are you new to Galaxy, or returning after a long time, and looking for help to get started? Take [a guided tour](https://proteomics.usegalaxy.eu/tours/core.galaxy_ui){:target="_blank"} through Galaxy's user interface.

**Want to learn more about proteomics? Check out the following lesson tutorials from the [Galaxy Training Network](https://galaxyproject.github.io/training-material/topics/proteomics/){:target="_blank"}:**

Lesson | Hands-on | Input dataset | Workflows | Galaxy tour
--- | --- | --- | --- | --- 
Detection and quantitation of N-termini degradomics via N-TAILS | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/ntails/tutorial.html){:target="_blank"}   |  | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/ntails/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/ntails/tours/){:target="_blank"}	 
Label-free versus Labelled - How to Choose Your Quantitation Method | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/labelfree-vs-labelled/tutorial.html){:target="_blank"} |  | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/labelfree-vs-labelled/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/labelfree-vs-labelled/tours/){:target="_blank"}
Mass spectrometry imaging 1: Loading and exploring MSI data | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/mass-spectrometry-imaging-loading-exploring-data/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1560645){:target="_blank"}
Metaproteomics tutorial | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.839701){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/metaproteomics/workflows/){:target="_blank"}
Peptide and Protein ID using OpenMS tools | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-id-oms/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/546301){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-oms/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-oms/tours/){:target="_blank"}
Peptide and Protein ID using SearchGUI and PeptideShaker | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-id-sg-ps/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/546301){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-sg-ps/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-id-sg-ps/tours/){:target="_blank"}
Peptide and Protein Quantification via Stable Isotope Labelling (SIL) | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/protein-quant-sil/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/1051552){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-quant-sil/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/protein-quant-sil/tours/){:target="_blank"}
Protein FASTA Database Handling | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/database-handling/tutorial.html){:target="_blank"} |  | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/database-handling/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/database-handling/tours/){:target="_blank"}
Proteogenomics 1: Database Creation | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-dbcreation/workflows/){:target="_blank"}
Proteogenomics 2: Database Search | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-dbsearch/workflows/){:target="_blank"}
Proteogenomics 3: Novel peptide analysis | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://doi.org/10.5281/zenodo.1489208){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/workflows/){:target="_blank"}
Secretome Prediction | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://zenodo.org/record/519260){:target="_blank"} | [<i class="fa fa-files-o" aria-hidden="true"></i>](https://zenodo.org/record/519260){:target="_blank"} | [<i class="fa fa-share-alt" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/secretome-prediction/workflows/){:target="_blank"} | [<i class="fa fa-magic" aria-hidden="true"></i>](https://github.com/galaxyproject/training-material/tree/master/topics/proteomics/tutorials/secretome-prediction/tours/){:target="_blank"}
{:.table.table-striped}

<br/>

# Gateways

Other specialized Galaxy servers curated by the Galaxy-P team containing tools, workflows, and hands-on/video tutorials.

Server | Tutorial(s) | Video
--- | --- | ---
[Proteogenomics Gateway](http://z.umn.edu/proteogenomicsgateway){:target="_blank"} | [Proteogenomics 1: Database creation](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.html){:target="_blank"}<br/>[Proteogenomics 2: Database search](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-dbsearch/tutorial.html){:target="_blank"}<br/>[Proteogenomics 3: Novel peptide analysis](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.html){:target="_blank"} | [![proteogenomics thumbnail](https://img.youtube.com/vi/Xwks5qKApj8/0.jpg)](https://www.youtube.com/watch?v=Xwks5qKApj8&t=4s){:target="_blank"}
[Metaproteomics Gateway](http://z.umn.edu/metaproteomicsgateway){:target="_blank"} | [Metaproteomics tutorial](https://galaxyproject.github.io/training-material/topics/proteomics/tutorials/metaproteomics/tutorial.html){:target="_blank"} | [![metaproteomics thumbnail](https://img.youtube.com/vi/Vywt_-D5Kek/0.jpg)](https://www.youtube.com/watch?v=Vywt_-D5Kek&t=1s){:target="_blank"}
{: .table.table-striped .tooltable}

<br/>

# Projects

Featured Galaxy-P tools supported with manuscripts, visualization plugins, docker containers, and hands-on tutorials.

Tool | Description | Paper | Tutorial | Docker
--- | --- | --- | --- | ---
CRAVAT-P | Stands for Cancer Related Analysis of Variants Toolkit for Proteogenomics. Submits, checks for, and retrieves data for cancer annotation | [Sajulga et. al 2018](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00404){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf){:target="_blank"} | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](http://z.umn.edu/gpcravatdocker){:target="_blank"}
QuanTP | Correlation between protein and transcript | [Kumar et. al 2019](https://pubs.acs.org/doi/10.1021/acs.jproteome.8b00727){:target="_blank"} | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.8b00404/suppl_file/pr8b00404_si_001.pdf){:target="_blank"} | [![docker-icon](https://cdn3.iconfinder.com/data/icons/social-media-2169/24/social_media_social_media_logo_docker-128.png)](http://z.umn.edu/quantpdocker){:target="_blank"}
MetaQuantome | Quantitative analysis of the function and taxonomy of microbiomes and their interaction |  | [<i class="fa fa-laptop" aria-hidden="true"></i>](https://galaxyproteomics.github.io/metaquantome_mcp_analysis/galaxy_tutorial/tutorial.html){:target="_blank"} |
{: .table.table-striped .tooltable}

<br/>

# News and Events
{% include home_carousel.html %}
{% include home_news_events.html %}



# Contributors

- [Bjoern Gruening](https://github.com/bgruening)
- [Helena Rasche](https://github.com/erasche)
- [Ray Sajulga](https://github.com/jraysajulga)
- [Subina Mehta](https://github.com/bebatut)

<br/>